templates/deployment-front.yaml

```diff
@@ -86,6 +86,14 @@ spec:
               value: {{ $value | quote }}
             {{- end }}
             {{- end }}
+            - name: COOKIE_USE_SSL
+            {{- if eq ((include "nlweb.helpers.getScheme" .) | toString) "https://" }}
+              value: "true"
+            {{- else }}
+              value: "false"
+            {{- end }}
+            - name: COOKIE_DOMAIN
+              value: {{ default ".neotys.com" (.Values.services.files.host | quote) }}
```

Comment:
Should be based on Values.services.webapp.host ?

Reply:
yes